src/BM2/SiteBundle/Controller/ActionsController.php

```diff
@@ -943,8 +943,8 @@ public function assignedAction(Request $request) {
 				if ($soldier->getCharacter() && $soldier->getCharacter()->isInBattle()) continue;
 				if ($soldier->getCharacter() && $soldier->getCharacter()->isDoingAction('military.regroup')) continue;
 				// FIXME: can't recall from your own estates or characters, but there should also be a message:
-				if ($soldier->getCharacter() && $soldier->getCharacter()->getUser() == $character->getUser()) continue;
-				if ($soldier->getBase() && $soldier->getBase()->getOwner() && $soldier->getBase()->getOwner()->getUser() == $character->getUser()) continue;
+				// if ($soldier->getCharacter() && $soldier->getCharacter()->getUser() == $character->getUser()) continue;
+				// if ($soldier->getBase() && $soldier->getBase()->getOwner() && $soldier->getBase()->getOwner()->getUser() == $character->getUser()) continue;
```

Comment:
Why comment this out? It is exploitable and was actively exploited, that is why it was added.